modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Defining a Custom Ingress Configuration
+
+[NOTE]
+====
+The tasks outline in this topic must be performed by an IT administrator
+====
+
+Runtime Fabric on Self-managed Kubernetes enables you to edit a template file that specifies a custom Ingress configuration. Using this file you can define annotations, rewrite rules, the ingressClassName, and more.
+
+The template file is a Kubernetes ConfigMap file. After customizing the template, you can use the `rtfctl apply` command-line utility to apply the custom Ingress configuration to your Kubernetes cluster.
+
+If you do not apply a custom template file, Runtime Fabric uses a default configuration that works with the out-of-the-box Kubernetes Ingress configuration.
+
+== Procedure
+
+. Copy the following into a text file:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingress-config
+  namespace: rtf
+data:
+  ingress.annotations: |
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/ssl-redirect: true
+    kubernetes.io/ingress.class: <name>
+  ingress.spec: |
+    ingressClassName: <name>
+    rules.http.paths.path: /${APP_PATH}
+----
+
+. Edit the text file as required.
++
+All changes should be made under the `data` section of this file. Do not modify any other sections of the template. The most common configuration options are.
++
+* `ingress.annotations`: Annotations defined in this section are added to the `annotations` section of every Ingress resource created of a Mule application. Each annotation must be defined on a new line. Annotations defined here are default annotations that you can override.
```

Comment:
`Annotations defined here are default annotations that you can override.` --> Message what we need to convey is:
"These Annotations defined in the template are default kubernetes annotations, which can be fully overridden by user, if they wish to. They are not obligated to carry these default values".

Also can we make these as bullet points? 
Annotations defined in this section are added to the `annotations` section of every Ingress resource created of a Mule application. 
- Each annotation must be defined on a new line. 
- Annotations defined in the template are default kubernetes annotations, which can be fully overridden by user, if they wish to. They are not obligated to carry these default values.

Reply:
This looks OK to me. @hegderohit89 Do you have strong preference with your wording?

Reply:
Not really, I am ok with the original text block too.  All i was saying is that we need to convey They are not obligated to carry these default values.

Reply:
Leaving the original text.